nomic/dataset.py

```diff
@@ -696,9 +696,9 @@ def _get_atoms(self, ids: List[str]) -> List[Dict]:
             raise ValueError("You must specify a list of ids when getting data.")
 
         response = requests.post(
-            self.project.atlas_api_path + "/v1/project/atoms/get",
-            headers=self.project.header,
-            json={'project_id': self.project.id, 'index_id': self.atlas_index_id, 'atom_ids': ids},
+            self.dataset.atlas_api_path + "/v1/project/atoms/get",
```

Comment:
I had forgot we had an endpoint literally called `get` that used `POST` instead of `GET` to retrieve data.